snapcraft/internal/parts.py

```diff
@@ -317,6 +317,9 @@ def build_env_for_part(self, part, root_part=True):
             env += dep_part.env(stagedir)
             env += self.build_env_for_part(dep_part, root_part=False)
 
+        env.append('PARALLEL_BUILD_COUNT={}'.format(
```

Comment:
since this method is called more than once (look at `for dep_part in part.deps`) can you put this inside the `if root_part` block?